<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvancedFilter</class>
 <widget class="QDockWidget" name="AdvancedFilter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>300</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>524287</width>
    <height>300</height>
   </size>
  </property>
  <property name="floating">
   <bool>false</bool>
  </property>
  <property name="features">
   <set>QDockWidget::AllDockWidgetFeatures</set>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Advanced filtering</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="advFilterLine1">
        <property name="text">
         <string>Filter : </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="filterLine1"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>on column</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="filterCombo1"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>then</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="filterLine2"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>on column</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="filterCombo2"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>then</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="filterLine3"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>on column</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="filterCombo3"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>148</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="filterLaunchButton">
        <property name="text">
         <string>Filter !</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>208</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
